packages/saber/vue-renderer/app/entry-server.js

```diff
@@ -19,12 +22,30 @@ export default async context => {
       get(obj, prop) {
         if (!head) {
           head = app.$meta().inject()
+        }else{
```

Comment:
I don't like how I am doing stuff this line down. I am searching for a better way